scalalib/src/mill/scalalib/Assembly.scala

```diff
@@ -13,13 +13,21 @@ import java.util.Collections
 import java.util.jar.JarFile
 import java.util.regex.Pattern
 import scala.jdk.CollectionConverters._
-import scala.tools.nsc.io.Streamable
 import scala.util.Using
 
 case class Assembly(pathRef: PathRef, addedEntries: Int)
 
 object Assembly {
 
+  private object Streamable {
+    def bytes(is: InputStream): Array[Byte] = {
+      val out = new java.io.ByteArrayOutputStream
+      IO.stream(is, out)
+      out.close()
+      out.toByteArray
+    }
+  }
+
```

Comment:
We're dropping Java 8 in Mill 0.12.0 and above, so I think this can be replaced with `is.readAllBytes()`